documentation/book/con-encryption-certificates.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
```

Comment:
This seems more like reference.

Reply:
This is a low priority user story and can be revisited after the beta release.

Reply:
Then we shouldn't include it right now. We don't want to do things wrong, it's better to leave it as it was before.